lib/symlink/index.js

```diff
@@ -2,16 +2,35 @@
 
 var through2 = require('through2');
 var fs = require('graceful-fs');
+var path = require('path');
+var valueOrFunction = require('value-or-function');
+
 var prepareWrite = require('../prepare-write');
+var defaultValue = require('../default-value');
+
+var boolean = valueOrFunction.boolean;
 
 function symlink(outFolder, opt) {
+  if (!opt) {
+    opt = {};
+  }
+
+  var rel = defaultValue(false, boolean(opt.relative));
```

Comment:
this should be inside the `linkFile` function and `boolean` should be passed `file` as the 2nd argument.